.github/workflows/cryptid-wasm.yml

```diff
@@ -0,0 +1,55 @@
+name: Bulding cryptid.wasm
+
+on: push
+
+jobs:
+  build:
+    name: build wasm
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: install clang
+      run: sudo bash -c "$(wget -O - https://apt.llvm.org/llvm.sh)"
+    - name: creating symlinks
+      run: |
+        sudo ln -svf /usr/bin/clang-9 /usr/bin/clang
+        sudo ln -svf /usr/bin/wasm-ld-9 /usr/bin/wasm-ld
+        sudo ln -svf /usr/bin/llc-9 /usr/bin/llc
+        sudo ln -svf /usr/bin/llvm-nm-9 /usr/bin/llvm-nm
+        sudo ln -svf /usr/bin/llvm-ar-9 /usr/bin/llvm-ar
+    - name: build wasi-libc
+      run: |
+        git clone https://github.com/WebAssembly/wasi-libc.git
+        cd wasi-libc
+        make WASM_CC=/usr/bin/clang WASM_AR=/usr/bin/llvm-ar WASM_NM=/usr/bin/llvm-nm
+        wget https://github.com/jedisct1/libclang_rt.builtins-wasm32.a
+        sudo mkdir /usr/lib/llvm-9/lib/clang/9.0.0/lib/wasi/
+        sudo mv libclang_rt.builtins-wasm32.a /usr/lib/llvm-9/lib/clang/9.0.0/lib/wasi/
+    - name: building warm
+      run: |
+        git clone https://github.com/cryptid-org/warm.git
+        cd warm/
+        ./make.sh clang "--sysroot=/home/runner/work/cryptid-native/cryptid-native/wasi-libc/sysroot -O2 -flto" llvm-ar
+    - name: building cryptid.wasm
+      run: |
+        git clone https://github.com/cryptid-org/cryptid-native.git
+        cd cryptid-native/
+        clang \
+          -O2 -flto -Wl,--lto-O2,--no-entry,--export=cryptid_ibe_bonehFranklin_setup,--export=cryptid_ibe_bonehFranklin_extract,--export=cryptid_ibe_bonehFranklin_encrypt,--export=cryptid_ibe_bonehFranklin_decrypt,--strip-all \
+          --target=wasm32-unknown-wasi \
+          --sysroot=/home/runner/work/cryptid-native/cryptid-native/wasi-libc/sysroot \
+          -o cryptid.wasm \
+          -Wl,--initial-memory=131072000 \
+          -I./include \
+          -I./dependencies/sha/include \
+          -I../warm \
+          ../warm/*.o \
+          ../warm/mpn/*.o \
+          ../warm/mpz/*.o \
```

Comment:
This is correct, however, we might be able to use the `.a` file generated by the `make.sh` in warm.

Reply:
Sure thing! :+1: 

I modified the necessary lines.